server/odc-core/src/main/java/com/oceanbase/odc/core/alarm/AlarmEvent.java

```diff
@@ -16,18 +16,20 @@
 
 package com.oceanbase.odc.core.alarm;
 
+import com.fasterxml.jackson.databind.JsonNode;
+
 import lombok.Getter;
 
 @Getter
 public class AlarmEvent {
 
     private final String eventName;
 
-    private final String eventMessage;
+    private final JsonNode eventMessage;
 
     private final AlarmLevel level;
 
-    public AlarmEvent(String eventName, String eventMessage, AlarmLevel level) {
+    public AlarmEvent(String eventName, JsonNode eventMessage, AlarmLevel level) {
```

Comment:
avoid involve Jackson model in AlarmEvent, use Map instead

Reply:
why I can not use JsonNode, I have filled all tool functions with JsonNode😂, I think `Map` and `JsonNode` functions are equivalent， because the jsonNode is third-party package class？

Reply:
given the dimensions struct of the last meeting design, which have nested properties, if I were to pull out `TaskType`, I think it would be easier for JsonNode to handle